src/commands/stac-setup/stac.setup.ts

```diff
@@ -0,0 +1,196 @@
+import { fsa } from '@chunkd/fs';
+import { command, flag, option, optional, string } from 'cmd-ts';
+import { StacCollection } from 'stac-ts';
+import ulid from 'ulid';
+
+import { CliInfo } from '../../cli.info.js';
+import { logger } from '../../log.js';
+import { slugify } from '../../utils/slugify.js';
+import { config, registerCli, tryParseUrl, UrlFolder, urlToString, verbose } from '../common.js';
+import { dataCategories } from './category.constants.js';
+
+export interface SlugMetadata {
+  geospatialCategory: string;
+  geographicDescription?: string;
+  region: string;
+  date: string;
+  gsd: string;
+}
+
+export interface StacCollectionLinz {
+  'linz:lifecycle': string;
+  'linz:geospatial_category': string;
+  'linz:region': string;
+  'linz:security_classification': string;
+  'linz:slug': string;
+  'linz:event_name'?: string;
+  'linz:geographic_description'?: string;
+}
+
+export const commandStacSetup = command({
+  name: 'stac-setup',
+  description: 'Setup STAC metadata for standardising workflows',
+  version: CliInfo.version,
+  args: {
+    config,
+    verbose,
+
+    addDateInSlug: flag({
+      // cmd-ts has a bug where defaultValue is always false and must be overridden
+      // To override: --add-date-in-slug=true
+      // Ref: https://github.com/Schniz/cmd-ts/issues/91
+      defaultValue: () => true,
+      long: 'add-date-in-slug',
+      description: 'Include the date in the linz:slug',
+      defaultValueIsSerializable: true,
+    }),
+
+    startDate: option({
```

Comment:
Does this have to be mandatory? I feel it should not if `addDateInSlug` is `false`?